app/src/components/BCIDView.tsx

```diff
@@ -111,8 +146,82 @@ const BCIDView: React.FC = () => {
     }
   }, [credentials]);
 
+  const cleanupAfterServiceCardAuthentication = (
+    status: AuthenticationResultType
+  ): void => {
+    InAppBrowser.closeAuth();
+
+    if (status === AuthenticationResultType.Cancel) {
+      setWorkflowInFlight(false);
+    }
+
+    if (navigation.canGoBack()) {
+      navigation.goBack();
+    }
+  };
+
+  const authenticateWithServiceCard = async (did: string): Promise<void> => {
+    try {
+      const url = `${Config.IDIM_PORTAL_URL}/${did}`;
+
+      console.log("target URL = ", url);
```

Comment:
should we get rid of this console statement here?

Reply:
Fixed.